cli/exchange/org.go

```diff
@@ -0,0 +1,165 @@
+package exchange
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/open-horizon/anax/cli/cliutils"
+	"github.com/open-horizon/anax/i18n"
+	"net/http"
+)
+
+type ExchangeOrgs struct {
+	Orgs      map[string]interface{} `json:"orgs"`
+	ListIndex int                    `json:"lastIndex"`
+}
+
+type OrgHeartbeat struct {
+	Adjust int `json:"intervalAdjustment"`
+	Max    int `json:"maxInterval"`
+	Min    int `json:"minInterval"`
+}
+
+type OrgExchangeReq struct {
```

Comment:
Please use Organization struct in the exchange package.

Reply:
The organization struct doesn't contain a field for Heartbeat. I also need them to have "omitempty" for the update function.

Reply:
Done. I updated the Organization struct to include a field for the Heartbeat.